mx.py

```diff
@@ -6384,7 +6384,7 @@ def _fetch(self, vcdir, repository=None, refspec=None, abortOnError=True, prune=
     def _log_changes(self, vcdir, path=None, incoming=True, abortOnError=True):
         out = OutputCapture()
         cmd = ['git', 'log', '{0}origin/master{1}'.format(
-                '..', '' if incoming else '', '..')]
+                '..', '' if incoming else '..')]
```

Comment:
format string takes 2 arguments but 3 were provided. Is my fix here correct?

Reply:
I've fixed this as follows:
```
        if incoming:
            cmd = ['git', 'log', '..origin/master']
        else:
            cmd = ['git', 'log', 'origin/master..']
```
It's also clearer.